README.md

```diff
@@ -1,2 +1,58 @@
-# indevelopme-coaching-publications
-These are the publications that players on our team created for publications.
+# indevelopme-coaching-publications  
+
+This repository holds a free collection of IT knowledge that has been created and reviewed for publication into Medium / Linkedin, and other platforms. Participants in the JustAskDavidB Coaching Program (aka GSD-IT) maintain this repository to showcase their IT compentency with hopes of improving IT literacy in non-IT communities. Highly consistent contributors have a chance to be offered paid opportunity which includes free self-training products such as Linkedin Learning, Coursera, etc. and opportunities to work on paid projects for inDevelopme. 
+
+## Purpose  
+
+The JustAskDavidB Coaching Program intends to help fives groups of people develop understanding of IT concepts at an accelerated rate:
+- individuals zero prior IT experience  
```

Comment:
- capitalize first word of all bulleted items
- insert "with" after "individuals"